metricq/agent.py

```diff
@@ -150,8 +150,19 @@ async def make_connection(self, url):
         else:
             ssl_options = None
 
+        client_properties = None
+        if connection_name:
+            # TODO remove outer client_properties with aiormq >= 5.1.1
```

Comment:
Do I see that right, the latest aio-pika mater requires aiormq < 4, while aiormq is at version 5.2.2?!
The comment-TODO should also be at the `setup.py` aio-pika requirement.

Reply:
Comment in setup.py is done in fd8689f126cf2f707d5a0977bd14a02f6f89649c